js/index.js

```diff
@@ -0,0 +1,36 @@
+const addToInputList = document.querySelector("#addName");
+const addButton = document.querySelector("#addNameToList");
+const addToOutputList = document.querySelector("#outputList");
+
+// Function to append 
+function addToList() {
+    const ol = addToOutputList.querySelector("ol");
+    const li = document.createElement("li");
+    const paragraph = document.createElement("p");
+
+    li.innerHTML = addToInputList.value;
+
+    if (!li.innerHTML) {
+        console.warn('Input text field cannot be empty');
```

Comment:
I like this extra validation to check that the input isn't empty.
However, using `console.warn` may not show the error to the users, we could have used [window.alert](https://developer.mozilla.org/en-US/docs/Web/API/Window/alert).

Reply:
Replacing the console warning with alert!

<img width="364" alt="image" src="https://user-images.githubusercontent.com/35523688/225375561-73f6192c-3570-461f-8ef7-6e925f709fb9.png">